operations/rates.md

```diff
@@ -472,36 +436,138 @@ Note that prices are defined daily, so when the server receives the UTC interval
 | `ClientToken` | string | required | Token identifying the client application. |
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
-| `RateId` | string | required | Unique identifier of the base [Rate](#rate) to update. |
-| `PriceUpdates` | array of [Price update](#price-update) | required, max 1000 items | Price updates. |
+| `Rates` | array of [Rate parameters](rates.md#rate-parameters) | required, max 1000 items | Information about rates to be created. |
+| `EnterpriseId` | string | optional | Unique identifier of the [Enterprise](enterprises.md#enterprise). Required when using a [Portfolio Access Token](../guidelines/multi-property.md), ignored otherwise. |
 
-#### Price update
+#### Rate parameters
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
-| `CategoryId` | string | optional | Unique identifier of the [Resource category](resources.md#resource-category) whose prices to update. If not specified, base price is updated. |
-| `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first time unit, in UTC timezone ISO 8601 format. See [Time units](../concepts/time-units.md).  |
-| `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last time unit, in UTC timezone ISO 8601 format. The maximum size of time interval depends on the service's time unit: 367 hours if hours, 367 days if days, or 60 months if months. See [Time units](../concepts/time-units.md). |
-| `Value` | number | optional | New value of the rate on the interval. If not specified, removes all adjustments within the interval. |
+| `ServiceId` | string | required | Unique identifier of the service. |
+| `RateGroupId` | string | required | Unique identifier of the rate group under which rate is assigned. |
+| `IsEnabled` | boolean | optional | Whether the rate is available to customers. `false` will be used as a default when not provided. |
+| `Type` | [Rate type](rates.md#rate-type) | required | Type of the rate. |
```

Comment:
This points to the enum that have 3 values but for adding we are allowing only public or private rate. Not availability block rate (that one is created only when you create availability block)